util/src/main/java/io/grpc/util/AdvancedTlsX509KeyManager.java

```diff
@@ -100,31 +99,44 @@ public String chooseEngineServerAlias(String keyType, Principal[] issuers,
    *
    * @param key  the private key that is going to be used
    * @param certs  the certificate chain that is going to be used
+   * @deprecated Use {@link #updateIdentityCredentials(X509Certificate[], PrivateKey)}
    */
+  @Deprecated
+  @InlineMe(replacement = "this.updateIdentityCredentials(certs, key)")
```

Comment:
Add the ExperimentalApi annotation to the deprecated methods so it is clear they can be deleted.

Reply:
Added in https://github.com/grpc/grpc-java/pull/11376